src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -737,6 +770,186 @@ end probability_measure -- namespace
 
 end probability_measure -- section
 
+section normalize_finite_measure
+/-! ### Normalization of finite measures to probability measures
+
+This section is about normalizing finite measures to probability measures.
+
+The weak convergence of finite measures to nonzero limit measures is characterized by
+the convergence of the total mass and the convergence of the normalized probability
+measures.
+-/
+
+namespace finite_measure
+
+variables {α : Type*} [measurable_space α] (μ : finite_measure α)
```

Comment:
you should probably make `[inhabited α]` as an assumption for the whole section. Also, since the outcome is not really relevant for the zero measure, I'd rather use `[nonempty α]` instead of `inhabited`.